packages/squidex/src/entities/crn/tutorials.ts

```diff
@@ -0,0 +1,15 @@
+import { Rest, Entity, Graphql } from '../common';
```

Comment:
is anything from this file being used anywhere at all? 

Those entities are in fact deprecated - the we don't do REST calls to Squidex anymore (not for fetching at least) and we generate the graphql types from the schema.

Reply:
Nice catch! They are not necessary, it was committed here https://github.com/yldio/asap-hub/pull/2143/commits/18eabc6a8af35dff0af7e42ac33c3f4908851ce3